lib/stdlib/doc/src/math.xml

```diff
@@ -102,9 +102,20 @@ erf(X) = 2/sqrt(pi)*integral from 0 to X of exp(-t*t) dt.</pre>
 
     <func>
       <name name="pi" arity="0" since=""/>
-      <fsummary>A useful number.</fsummary>
+      <fsummary>Ratio of circumfence to diameter of a circle.</fsummary>
```

Comment:
```suggestion
      <fsummary>Ratio of the circumference of a circle to its diameter.</fsummary>
```

Reply:
At the same time, if you don't want to accept the suggestion, I point out that "circumfence" is a typo.